core/src/modules/ops/recordProperty.ts

```diff
@@ -199,6 +199,17 @@ export namespace RecordPropertyOps {
 }
 
 // utilities (private)
+async function updateIds(ids: string[], values: { [key: string]: any }) {
+  const max = config.batchSize.internalWrite;
```

Comment:
I think this is the only place that uses the new `GROUPAROO_INTERNAL_BATCH_SIZE` ENV.  I don't **think** this would be a limiting problem necessarily because it's just one small-ish statement (UPDATE recordProperties where ID in ...).

That said, this will allow us to play with this setting and try out new values, so it's a 👍 from me!

Reply:
Also, this is a nice utility method

Reply:
The problem the customer was seeing is that the size of this is variable on the number of properties that can be batched together, so it got to be quite large (100k) on a wide table and froze things up.

Reply:
I assume we'll find other places to use it, roughly as a way to think about how much simultaenous updating the database can do.